tests/testAutopas/tests/selectors/tuningStrategy/PredictiveTuningTest.cpp

```diff
@@ -1,329 +0,0 @@
-/**
- * @file PredictiveTuningTest.cpp
```

Comment:
This test was not deleted but moved. However, since too much changed, git considers this a delete + new file.